08smart-contracts-vyper.asciidoc

```diff
@@ -184,6 +184,17 @@ As you can see, the conversion process ensures that no information can be lost;
 
 Choosing explicit over implicit typecasting means that the developer is responsible for performing all casts. While this approach does produce more verbose code, it also improves the safety and auditability of smart contracts.
 
+==== Infinite Loop
+((("infinite loop")))((("Solidity","infinite loop")))((("Vyper","infinite loop")))Although there is no merit because of gaslimit, developers can write an endless loop processing in Solidity. Infinite loop makes it impossible to set an upper bound on gas limits, opening the door for gas limit attacks. Therefore, Vyper doesn't permit you to write the processing and has the following three features:
+
+The while statement::
+you can use while statement in Solidity, but Vyper doesn't have the statement.
```

Comment:
I guess it's better to mark the keywords? for example, `for` instead of for, `while` instead of while.

Reply:
Thanks ! I've fixed. 
1aa1e53